certify-service/src/main/java/io/mosip/certify/services/StatusListCredentialService.java

```diff
@@ -170,8 +170,9 @@ public StatusListCredential generateStatusListCredential(String statusPurpose) {
 
             // Sign the status list credential
             Map<String, String> signerSettings = new HashMap<>();
-            signerSettings.put(Constants.APPLICATION_ID, CertifyIssuanceServiceImpl.keyChooser.get(vcSignAlgorithm).getFirst());
-            signerSettings.put(Constants.REFERENCE_ID, CertifyIssuanceServiceImpl.keyChooser.get(vcSignAlgorithm).getLast());
+            signerSettings.put(Constants.APPLICATION_ID, CertifyIssuanceServiceImpl.keyChooser.get(vcSignCryptoSuite).getFirst());
+            signerSettings.put(Constants.REFERENCE_ID, CertifyIssuanceServiceImpl.keyChooser.get(vcSignCryptoSuite).getLast());
+            signerSettings.put(Constants.VC_SIGN_CRYPTO_SUITE, vcSignCryptoSuite);
```

Comment:
I was not able to understand why vcSignCryptoSuite is used in status list also.
I would like to known if you have 2 different credential types with 2 different vcSignCryptoSuite configured, you will use 2 different statuslist credentials ?
cc: @likhitharl